### PR TITLE
Create ui-c8y-1020-0-0-services-removed-from-the-app-module-and-loaded-as-a-plugin.md

### PR DESCRIPTION
**Pull request by gen-App-Enablement**
Release note created by dafk

Ticket: [DM-2973](https://cumulocity.atlassian.net/browse/DM-2973)
Original PR: [4897](https://github.softwareag.com/IOTA/cumulocity-ui/pull/4897)

### CODE REVIEW

File: content/change-logs/device-management/ui-c8y-1020-0-0-services-removed-from-the-app-module-and-loaded-as-a-plugin.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component--KIsStyzM
+    label: Device Management app
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: DM-2973
+version: 1020.0.0
+---
+Services removed from the app module and loaded as a plugin. (#4897)
```
Comment (dafkk) [inline content/change-logs/device-management/ui-c8y-1020-0-0-services-removed-from-the-app-module-and-loaded-as-a-plugin.md:17]:
```suggestion
The services component was previously an integral part of the default Device Management application. In this update, the services component has been restructured to function as a plugin. It remains included by default but now offers the flexibility to be added or removed as needed. (#4897)
```

File: content/change-logs/device-management/ui-c8y-1020-0-0-services-removed-from-the-app-module-and-loaded-as-a-plugin.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component--KIsStyzM
+    label: Device Management app
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: DM-2973
+version: 1020.0.0
+---
+The services component was previously an integral part of the default Device Management application. In this update, the services component has been restructured to function as a plugin. It remains included by default but now offers the flexibility to be added or removed as needed. (#4897)
```
Comment (BeateRixen) [inline content/change-logs/device-management/ui-c8y-1020-0-0-services-removed-from-the-app-module-and-loaded-as-a-plugin.md:17]:
```suggestion
The services component was previously an integral part of the default Device Management application. With this change, the services component has been restructured to function as a separate plugin. It remains included by default but now offers the flexibility to be added or removed as needed. 
```

File: content/change-logs/device-management/ui-c8y-1020-0-0-services-removed-from-the-app-module-and-loaded-as-a-plugin.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: Services removed from the app module and loaded as a plugin. (#4897)
```
Comment (BeateRixen) [inline content/change-logs/device-management/ui-c8y-1020-0-0-services-removed-from-the-app-module-and-loaded-as-a-plugin.md:3]:
```suggestion
title: Services component extracted as separate plugin
```

**Review by BeateRixen — APPROVED** [review]

**Review by bozhidardedov — APPROVED** [review]

**Comment by BeateRixen** [comment]
@dafkk FYI. For feature entries please have the PM, in this case @aaronraab approve the PR as well. As he is on PTO, it is ok for now :-).

**Pull request merged**